app/src/main/java/org/sopt/soptseminar_week1/api/GithubService.kt

```diff
@@ -3,19 +3,20 @@ package org.sopt.soptseminar_week1.api
 import org.sopt.soptseminar_week1.data.GithubRepositoryInfo
 import org.sopt.soptseminar_week1.data.GithubUserInfo
 import retrofit2.Call
+import retrofit2.Response
 import retrofit2.http.GET
 import retrofit2.http.Path
 
 interface GithubService {
     @GET("/users/{username}/repos")
-    fun getRepositories(@Path("username") username: String): Call<List<GithubRepositoryInfo>>
+    suspend fun getRepositories(@Path("username") username: String): Response<List<GithubRepositoryInfo>>
```

Comment:
```suggestion
    suspend fun getRepositories(@Path("username") username: String): List<GithubRepositoryInfo>
```

suspend modifier를 사용할 수 있으면 위와 같이 작성할 수 있어
근데 우리 팀이 모두다 suspend modifier를 사용하는건 아니니까 GithubService에서 suspend를 사용하기보다 Call을 사용해서 작성하는게 조금 더 좋을 수도 있어. 물론 이 방식이 나쁘다는건 아니야, 굉장히 잘했어